src/components/MessagesList/MessagesList.vue

```diff
@@ -932,24 +934,25 @@ export default {
 				return
 			}
 
-			this.setChatScrolledToBottom(false)
```

Comment:
After swtiching to conversation, handleScroll is triggered. Regardless of container height, it sets value to false (whereas it shouldn't for non-scrollable chats)
For scrollable chats it also shouldn't set to false, when receiiving message (handleScroll is triggered with 'down' direction)

Reply:
Modify to set 'false' only if we're above 10px from bottom. So if we're sticky, that shouldn't cause button appearance